fe/fe-core/src/main/java/org/apache/doris/load/loadv2/BrokerLoadJob.java

```diff
@@ -364,9 +364,16 @@ private void updateLoadingStatus(BrokerLoadingTaskAttachment attachment) {
 
     @Override
     public void updateProgress(Long beId, TUniqueId loadId, TUniqueId fragmentId, long scannedRows,
-                               long scannedBytes, boolean isDone) {
-        super.updateProgress(beId, loadId, fragmentId, scannedRows, scannedBytes, isDone);
-        progress = (int) ((double) loadStatistic.getLoadBytes() / loadStatistic.totalFileSizeB * 100);
+                               long scannedBytes, long readBytes, boolean isSupportReadBytes, boolean isDone) {
+        super.updateProgress(beId, loadId, fragmentId, scannedRows, scannedBytes,
+                readBytes, isSupportReadBytes, isDone);
+
```

Comment:
we'd better show ReadBytes alongside LoadBytes in `show load` results.

Reply:
> we'd better show ReadBytes alongside LoadBytes in `show load` results.

add those fields in `getShowInfo` of  `loadJob.java` V2, is this right?